buildSrc/build.gradle.kts

```diff
@@ -1,16 +1,15 @@
 import org.gradle.kotlin.dsl.`kotlin-dsl`
 
-buildscript {
-  repositories {
-    mavenCentral()
-  }
-}
-
 repositories {
-  mavenCentral()
+  mavenLocal()
```

Comment:
My reasoning for removing this was.. All dependencies used in `buildSrc` refer to gradle plugins which are found in plugin portal normally, unless we decide to use some internal plugin, in which case it might be desirable to resolve it from `mavenLocal`. Have we had cases were we use normal dependencies in `buildSrc` previously?

Reply:
Nop that make sense :)